dbt/adapters/athena/impl.py

```diff
@@ -83,8 +83,8 @@ class AthenaConfig(AdapterConfig):
         table_properties (str) : Table properties to add to the table, valid for Iceberg only.
         native_drop (str) :  Relation drop operations will be performed with SQL, not direct Glue API calls.
         seed_by_insert (bool) : default behaviour uploads seed data to S3.
-        lf_tags_config (Dict[str, Any]) : AWS lakeformation tags to associate with the table and columns.
-        seed_s3_upload_args (Dict[str, Any]) : Dictionary containing boto3 ExtraArgs when uploading to S3.
+        lf_tags_config (dict[str, Any]) : AWS lakeformation tags to associate with the table and columns.
+        seed_s3_upload_args (dict[str, Any]) : Dictionary containing boto3 ExtraArgs when uploading to S3.
```

Comment:
do we still need typing in docstrings as they are already present in args list ? I believe we can remove typing from docstring. WDYT ?

Reply:
Yes we can remove typing from doctring indeed